### ISSUE TITLE
calc() in media query reported as parse error

### ISSUE THREAD

**Issue by dd8** [w3c/css-validator#194]
```
@media screen and (max-width: calc(2000px-1px)) { 
  .col { width: 200px; }
}
```

Spec says calc() can be used wherever a length can be used
https://drafts.csswg.org/css-values-3/#calc-notation

... and Chrome/Firefox have implemented this
https://developer.mozilla.org/en-US/docs/Web/CSS/calc#Browser_compatibility

See also:
https://stackoverflow.com/questions/23667208/calc-not-working-within-media-queries

**Comment by ylafon** [comment]
calc(2000px-1px) is invalid, per https://www.w3.org/TR/css-values-3/#calc-syntax

> In addition, white space is required on both sides of the + and - operators. (The * and / operaters can be used without white space around them.)

**Comment by ylafon** [comment]
Closed with the merge of #195
